SECURITY/SECURITY/main.asm

```diff
@@ -0,0 +1,61 @@
+;
+; SECURITY.asm
+;
+; Created: 10/29/2018 9:14:32 AM
+; Author : PC
+;
+; TRIGGER = 0 (PORTJ) , ECHO = 0 (PORT K), LIGHT = ANY OF THE PORT B () 
+ldi r25, 0xFF; preloading IO mode to output
```

Comment:
When using a register for the same purpose for the entirety of the program you could have considered to redefine its name to make it easier to read your code. `.def HIGH_VALUE = r25` then for instance line 26 would be `sts portj, HIGH_VALUE`.